fabric-networking-v0/src/main/java/net/fabricmc/fabric/api/server/PlayerStream.java

```diff
@@ -43,30 +43,24 @@
 	private PlayerStream() { }
 
 	public static Stream<ServerPlayerEntity> all(MinecraftServer server) {
-		if (server.getPlayerManager() != null) {
-			return server.getPlayerManager().getPlayerList().stream();
-		} else {
-			return Stream.empty();
-		}
+		return server.getPlayerManager() != null ? server.getPlayerManager().getPlayerList().stream() : Stream.empty();
 	}
 
-	public static Stream<PlayerEntity> world(World world) {
+	public static Stream<? extends PlayerEntity> world(World world) {
 		if (world instanceof ServerWorld) {
-			// noinspection unchecked
-			return ((Stream) ((ServerWorld) world).getPlayers().stream());
+			return (((ServerWorld) world).getPlayers().stream());
 		} else {
-			throw new RuntimeException("Only supported on ServerWorld!");
+			throw new UnsupportedOperationException("Only supported on ServerWorld!");
```

Comment:
Good catch here, `UnsupportedOperation` is more explicit in why this could have failed.